src/patterns/molecules/forms/status.hbs

```diff
@@ -0,0 +1,107 @@
+---
+name: Status
+description: More status messages
```

Comment:
This description probably needs to be fleshed out more. But this looks like work in progress so I'm not sure I should add any more comments yet.